### DISCUSSION TITLE
Constraining particles to a shape/container

### DISCUSSION THREAD

**Discussion by nebrekab** [matteobruni/tsparticles#5033] [General]
Hi there,

Is it possible to constrain the particles with the boundaries of a shape? For example have them bounce around circular bounds or a custom shape area? 

Thanks!

**Comment by matteobruni** [comment]
No it's not possible, the container is always considered as a rectangular area.

**Reply by nebrekab** [reply]
Ok, fair enough.

Thanks for the swift reply.
